ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -324,9 +324,12 @@ def delete_buckets(bucket_prefix, hours):
     aws = AWS()
     buckets_to_delete = aws.get_buckets_to_delete(bucket_prefix, hours)
     logger.info(f"buckets to delete: {buckets_to_delete}")
+    filename = "s3_bucket_deletion_list.txt"
+    with open(filename, "w") as f:
```

Comment:
should we mabye use `"a"`
To append to file?

Not sure if this function will be called multiple time for some reason in future you will overwrite what was already written in file.

Reply:
I copied the code from here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/cleanup/aws/cleanup.py#L458

Reply:
CC @clacroix12